nodejs/lesson03/solution/hello.js

```diff
@@ -19,9 +19,9 @@ function sayHello(helloTo) {
             span.finish();
         })
         .catch( err => {
-            span.setTag(Tags.ERROR, true)
```

Comment:
why remove this one?

Reply:
Just to be consistent with other language implementations. We are not marking error on parent spans but only child.

Reply:
I don't think it's correct. Error here indicates that the operation failed (may not even have made the call to the server), so the span is in error

Reply:
If the request fails during the call to server, this code will mark both parent and child erroneous. Are you suggesting that?

Reply:
I think that's correct. Don't think of a child span here (it's an implementation detail of format_string  and print_hello functions), think about the `say-hello` span - if it gets an exception from its dependencies, then it cannot succeed and therefore should be marked as failed.

Reply:
Ok, I am reverting my code. I think that all language implementations should be consistent, either way, it should be across all exercises. We never mark parent span erroneous in others implementations. 

The other way I could think of is: stop throwing error up the chain in format-string, and print_hello function. That will never mark sayHello function erroneous, unless it is.

Marking the parent as of now, doesn't sound right to me.